ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala

```diff
@@ -150,21 +152,9 @@ private[apiserver] final class ApiSubmissionService private[services] (
         logger.debug("Success")
         Success(())
 
-      case Success(Overloaded) =>
-        logger.info("Back-pressure")
-        Failure(Status.RESOURCE_EXHAUSTED.asRuntimeException)
-
-      case Success(NotSupported) =>
```

Comment:
Have you checked that these errors will not be reported by canton or corda?

Reply:
These errors no longer exist in the v2 enumeration, so they can't be any more. We can re-introduce them if a ledger driver has problems without them, but I expect it'll be fine.